chaste_codegen/_command_line_script.py

```diff
@@ -5,98 +5,161 @@
 import os
 from collections import OrderedDict
 
-import cellmlmanip
-
 import chaste_codegen as cg
 from chaste_codegen._script_utils import write_file
+from chaste_codegen.model_with_conversions import load_model_with_conversions
 
 
 # Link names to classes for converting code
-TRANSLATORS = OrderedDict([('Chaste', cg.NormalChasteModel),
-                           ('ChasteOpt', cg.OptChasteModel),
-                           ('CVODE', cg.CvodeChasteModel),
-                           ('CVODEWithDataClamp', cg.CvodeWithDataClampModel),
-                           ('BackwardsEuler', cg.BackwardEulerModel),
-                           ('RushLarsen', cg.RushLarsenModel),
-                           ('RushLarsenOpt', cg.RushLarsenOptModel),
-                           ('GeneralisedRushLarsen1', cg.GeneralisedRushLarsenFirstOrderModel),
-                           ('GeneralisedRushLarsen1Opt', cg.GeneralisedRushLarsenFirstOrderModelOpt),
-                           ('GeneralisedRushLarsen2', cg.GeneralisedRushLarsenSecondOrderModel),
-                           ('GeneralisedRushLarsen2Opt', cg.GeneralisedRushLarsenSecondOrderModelOpt)])
+TRANSLATORS = OrderedDict(

```

Comment:
I'd add a comment about what the fields in the tuples represent!

Reply:
yes good idea thanks!